src/corehost/cli/args.cpp

```diff
@@ -61,8 +61,19 @@ bool parse_arguments(
     const int argc, const pal::char_t* argv[], arguments_t* arg_out)
 {
     arguments_t& args = *arg_out;
+
+    // Try to use argv[0] as own_path to allow for hosts located elsewhere
+    if (argc >= 1)
+    {
+        args.own_path = argv[0];
+        if (!pal::realpath(&args.own_path))
+        {
+            args.own_path.clear();
```

Comment:
Do we want to trace this situation, so we know it happened?

Reply:
I added a trace. Also checked to see if the actual file exists to prevent any backwards compatibility issues (or forward compatibility issues if the file ends up being used for something).